SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialVirtualWorkerTranslator.cpp

```diff
@@ -231,27 +255,33 @@ void SpatialVirtualWorkerTranslator::QueryForWorkerEntities()
 // returned information will be thrown away.
 void SpatialVirtualWorkerTranslator::WorkerEntityQueryDelegate(const Worker_EntityQueryResponseOp& Op)
 {
+	bWorkerEntityQueryInFlight = false;
+
 	check(StaticComponentView.IsValid());
 	if (!StaticComponentView->HasAuthority(SpatialConstants::INITIAL_VIRTUAL_WORKER_TRANSLATOR_ENTITY_ID, SpatialConstants::VIRTUAL_WORKER_TRANSLATION_COMPONENT_ID))
 	{
 		UE_LOG(LogSpatialVirtualWorkerTranslator, Log, TEXT("(%s) Received response to WorkerEntityQuery, but don't have authority over VIRTUAL_WORKER_MANAGER_COMPONENT.  Aborting processing."), *WorkerId);
+		return;
 	}
 	else if (Op.status_code != WORKER_STATUS_CODE_SUCCESS)
 	{
-		UE_LOG(LogSpatialVirtualWorkerTranslator, Log, TEXT("(%s) Could not find Worker Entities via entity query: %s"), *WorkerId, UTF8_TO_TCHAR(Op.message));
-	}
-	else if (Op.result_count == 0)
```

Comment:
I get it _shouldn't_ happen ever since we're running the query from an active worker but does that make it totally not useful to have?

Reply:
I see the additional log added at the bottom of the function but I think "not finding any worker entities via the query" is a more precise description than "waiting for all virtual workers to be assigned" which implies some worker entities might've been found.

Reply:
I actually disagree on this one. I feel like the code is simpler without the check. It's not something that should ever happen, and the system will still behave correctly even if it does, so I'd rather not clutter the code.